benchmark/micro_benchmark_utils.cpp

```
@@ -23,17 +23,18 @@ namespace unodb::benchmark {
 // Teardown
 
 template <class Db>
-void destroy_tree(Db &db, ::benchmark::State &state) {
+void destroy_tree(Db &instance, ::benchmark::State &state) {
```

Cppcheck (2.10-2)

[style] The function 'destroy_tree' is never used.

(unusedFunction)